frontend/src/views/Tables/TablePreview.js

```diff
@@ -4,7 +4,7 @@ import { Card, CircularProgress } from '@mui/material';
 import * as PropTypes from 'prop-types';
 import { DataGrid } from '@mui/x-data-grid';
 import { styled } from '@mui/styles';
-import previewTable from '../../api/DatasetTable/previewTable';
```

Comment:
The file being referenced does not exist

Reply:
This issue has also been addressed